src/tooling/piral-cli/src/common/http.ts

```diff
@@ -77,6 +78,14 @@ export function postFile(target: string, key: string, file: Buffer, ca?: Buffer)
           const { data, statusText, status } = error.response;
           const message = getMessage(data);
           log('unsuccessfulHttpPost_0066', statusText, status, message || '');
+        } else if (error.isAxiosError) {
+          // axios initiated error: try to parse message from error object
+          let errorMessage: string = error.errno || 'Unknown Axios Error';
+          if (error.toJSON) {
+            const errorObj: Object = error.toJSON();
```

Comment:
Not sure but I guess `toJSON` returns `any`? In this case I would use `{ message?: string }` instead of `Object`; then the `errorObj["message"]` expressions could be replaced by `errorObj.message`, i.e., we could end up with

```ts
errorMessage = errorObj?.message ?? errorMessage;
```

Reply:
Good suggestion, done!